### ISSUE TITLE
Checksum mismatch for PyPy 3.6-7.1.0 for macOS

### ISSUE THREAD

**Issue by anton-petrov** [pyenv/pyenv#1316]
This is correct checksum for _macOS_ for _PyPy 3.6-7.1.0_ archive!

`d46e005ba095cb4a7006079ffbf4fe63c18cf5e9d8ce9ce8383efc1a4863ab5b`

Already installed with this checksum, all fine :)

**Comment by joshfriend** [comment]
ffs, why do they do this to me?
 
I _know_ i had this one right when i pushed the first commit. I run mac os and always test those. In fact, I even have the tarball i downloaded to verify the checksum against the pypy downloads page: 

```
$ shasum -a 256 ~/Downloads/pypy3.6-v7.1.0-src.tar.bz2
3e3091798cff9e7b6de35f7aa7f71965e06dcbf35112eba202d4ded7675f95f4  /Users/josh/Downloads/pypy3.6-v7.1.0-src.tar.bz2
```

**Comment by anton-petrov** [comment]
This is really strange 😼

**Comment by anton-petrov** [comment]
```
Antons-MacBook-Pro:Downloads anton$ shasum -a 256 pypy3.6-v7.1.0-osx64.tar.bz2
**d46e005ba095cb4a7006079ffbf4fe63c18cf5e9d8ce9ce8383efc1a4863ab5b**  pypy3.6-v7.1.0-osx64.tar.bz2
```

**Comment by joshfriend** [comment]
I checked all the 3.6-7.1.0 files this time, when #1314 was reported i only fixed that one specific file. Turns out they were all updated.

**Comment by anton-petrov** [comment]
Ok, I saw that:) thanks, you are really fast!

**Comment by mattip** [comment]
PyPy release manager here. I moved the release tag before the release was final. While we will change our release process to not move tags anymore, I would suggest whatever tool is used to pull these tarballs have some kind of delay of a day or two before it decides to package new releases, or even better wait for an official announcement by projects before packaging new releases.

**Comment by mattip** [comment]
... although I guess it is difficult to decide what "an official annoucement" is. Again, I apologize for the confusion, will try to change our process accordingly.

**Comment by joshfriend** [comment]
Hi @mattip, pulling new pythons into pyenv is currently not an automated process. Maintainers such as myself just keep an eye out for "official announcements" or issues posted on github by our users to alert us to new releases that we should pull in. In the case of PyPy, I found out about it from [your post about it on HN](https://news.ycombinator.com/item?id=19478177). That post linked to the official pypy blog so I think I correctly assumed it to be official. I guess maybe there was some unfortunate timing with the release announcement.

**Comment by mattip** [comment]
Looking back at the timeline you are correct and this was totally my mistake. I should have released a 7.1.1 and not moved the 7.1.0 tag after the release. Again, I apologize.

**Comment by joshfriend** [comment]
Thanks for the clarification. Looking forward to trying out the new release 😄
